src/appleseed.studio/mainwindow/renderingsettingswindow.cpp

```diff
@@ -707,6 +747,35 @@ namespace
 
             set_config(config, param_path, max_path_length);
         }
+
+        void load_separate_bounce_settings(
+            const Configuration&    config,
+            const string&           widget_key_prefix,
+            const string&           param_path,
+            const string&           type)
+        {
+            const size_t DefaultMaxBounces = 8;
+
+            const size_t max_path_length =
```

Comment:
This variable should use the word `bounce` instead of `path_length`. See another comment I made in this review: bounces and path lengths are different things.